.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
Curious, why the preference to omit semicolons? (I'm not necessarily against it)

Reply:
Actually, I don't prefer that, but I see this frontend side so I want to be consistent. Tell me what you want and I'll be ok !

Reply:
Ah, ok! I actually would prefer semicolons on both front and back-end

Reply:
Ok for me, I update it as soon as possible.

Reply:
Done